src/panels/lovelace/cards/hui-entities-card.ts

```diff
@@ -197,7 +197,10 @@ class HuiEntitiesCard extends LitElement implements LovelaceCard {
   }
 
   private renderEntity(entityConf: EntitiesCardEntityConfig): TemplateResult {
-    const element = createRowElement(entityConf);
+    const element = createRowElement({
```

Comment:
Let's only make a copy of `entityConf` if `this._config.state_color` is `true` to avoid creating more objects than we need.